tests/python/kaolin/spc/compare.py

```diff
@@ -0,0 +1,198 @@
+import math
```

Comment:
What is this python module doing? It's not a test file so it doesn't belong here.